conf/config.properties

```diff
@@ -19,40 +19,39 @@ DB_URL=http://192.168.8.2:6666
 #TimescaleDB
 #FakeDB
 DB_SWITCH=IoTDB
-#Database name, currently required in InfluxDB and TimescaleDB
+#Database name, only required for InfluxDB and TimescaleDB
 DB_NAME=test
-### benchmark运行模式选择参数 ###
+### benchmark work mode ###
 #目前支持多种运行模式，分别为：
 #testWithDefaultPath--常规测试模式，支持多种读和写操作的混合负载
 #writeWithRealDataSet--写入真实数据集模式，需要配置 FILE_PATH 以及 DATA_SET
 #queryWithRealDataSet--查询真实数据集模式，需要配置REAL_QUERY_START_TIME, REAL_QUERY_STOP_TIME, DATA_SET 以及 testWithDefaultPath模式下查询有关参数
-#insertTestWithDefaultPath--重构前的纯写入模式
-#queryTestWithDefaultPath--重构前的纯查询模式
-#insertTestWithUserDefinedPath--样例数据生成模式
 #serverMODE--服务器资源使用监控模式（该模式下运行通过ser-benchmark.sh脚本启动，无需手动配置该参数）
 #importDataFromCSV--从CSV文件中读取数据模式
-#executeSQLFromFile--从文件中读取SQL语句并依次一个一个提交执行
-BENCHMARK_WORK_MODE=insertTestWithDefaultPath
-#各操作的比例,按照顺序 写:Q1:Q2:Q3:Q4:Q5:Q6:Q7:Q8 ,Qi表示QUERY_CHOICE=i时的查询操作。请注意使用英文冒号。比例中的每一项是number类型。
+BENCHMARK_WORK_MODE=testWithDefaultPath
+#各操作的比例,按照顺序 写入:Q1:Q2:Q3:Q4:Q5:Q6:Q7:Q8, 请注意使用英文冒号。比例中的每一项是整数。
+#Qi表示的查询如下:
+#Q1--精确点查询 select v1... from data where time = ? and device in ?
+#Q2--范围查询（只限制起止时间）select v1... from data where time > ? and time < ? and device in ?
+#Q3--带值过滤的范围查询 select v1... from data where time > ? and time < ? and v1 > ? and device in ?
+#Q4--带时间过滤的聚合查询 select func(v1)... from data where device in ? and time > ? and time < ?
+#Q5--带值过滤的聚合查询 select func(v1)... from data where device in ? and value > ?
+#Q6--带值过滤和时间过滤的聚合查询 select func(v1)... from data where device in ? and value > ? and time > ? and time < ?
+#Q7--分组聚合查询（GROUP BY）（暂时只能生成有一个时间区间的语句）
+#Q8--最近点查询 select time, v1... where device = ? and time = max(time)
 OPERATION_PROPORTION=2:0:0:0:0:0:0:0:0
-#启动测试前是否清除旧数据(是否执行cleanup)
+#启动测试前是否清除旧数据(是否执行cleanup) 对IoTDB除外 因为IoTDB不支持删除Storage Group
```

Comment:
now it seems that IoTDB has delete SG function using session. However, we can not delete SG using jdbc.